config/index.js

```diff
@@ -150,6 +150,13 @@ var conf = convict({
       env: 'VERIFY_URL',
       arg: 'verify-url'
     },
+    verificationSigninUrl: {
```

Comment:
See [this comment in the auth-mailer](https://github.com/mozilla/fxa-auth-mailer/pull/141/files#r59712814) about changing the name of this variable to `tokenVerifyUrl`.